src/main/java/hek/models/EventType.java

```diff
@@ -13,7 +13,11 @@
      */
     AR, CE, CD, CH, CW, FI, FE, FA, FL, LP, OS, SS, EF, CJ, PG, OT, NR, SG, SP, CR, CC, ER, TOB, HY, BU, EE, PB, PT;
 
-
+    /**
+     * Uses String representation to wrap into the EventType class object.
+     * @param s - string representation
+     * @return EventType object
```

Comment:
FYI: We had to do this because "TO" is reserved word in PostgreSQL and we were not able to create table named "TO". See: https://www.postgresql.org/docs/7.3/static/sql-keywords-appendix.html